test/EVMHost.h

```diff
@@ -44,6 +46,8 @@ class EVMHost: public evmc::MockedHost
 	/// afterwards.
 	static evmc::VM& getVM(std::string const& _path = {});
 
+	static bool checkVmPaths(std::vector<boost::filesystem::path> const& _vmPaths);
```

Comment:
Please document that it returns true if and only if an EVM VM was found (ignoring ewasm failures) and throws if multiple VMs of the same type are provided.